include/plat/pc99/plat/32/plat_mode/machine/hardware.bf

```diff
@@ -45,4 +45,23 @@ block vtd_pte {
     field       read                1
 }
 
+-- Intel VT-d Interrupt Remapping Table Entry for Remapped Interrupts
```

Comment:
The "for Remapped Interrupts" part is not necessary, that's obvious because it's the interrupt remapping table.

Reply:
The Interrupt Remapping Table also supports posted interrupts, and we are planning to add support to posted interrupts.

Reply:
Posted interrupts are a way to inject IRQs into a VM running on another CPU core, and they're partially configured with this table. Whether interrupts are remapped or posted is configured independently, but practically you also want posted interrupts to be remapped.

But they used previously reserved bits to configure posting, so the entry format will be the same for either type. Then the "for Remapped Interrupts" will become "for remapped and posted interrupts", which also seems unnecessary verbose to me.

Reply:
The IRTE structures for remapped interrupts and posting interrupts are slightly different according to the spec, the one for posted interrupts has extra fields for Posted Descriptor Address and other stuff.